docs/contributing.rst

```diff
@@ -96,6 +96,26 @@ systems, ``evmone.dll`` on Windows systems and ``libevmone.dylib`` on macOS.
 On macOS some of the testing scripts expect GNU coreutils to be installed.
 This can be easiest accomplished using Homebrew: ``brew install coreutils``.
 
+Search paths
+------------
+
+The dependent libraries (such as ``evmone.so`` / ``evmone.dll``) can be
+located in different locations to be found automatically.
+
+The paths that are looked up in order are:
+
+- ``$ETH_EVMONE``
+- ``$PROJECT_ROOT/deps``
+- ``$PROJECT_ROOT/deps/lib``
+- ``$PROJECT_ROOT/../deps``
+- ``$PROJECT_ROOT/../deps/lib``
+- ``$PROJECT_ROOT/../../deps/lib``
+- ``$PROJECT_ROOT/../../deps/lib``
+- ``$PROJECT_ROOT/../../../deps/lib``
```

Comment:
It will also try CWD.
Also, if the library is not found in any of these, we still let the system look up the dynamic library in the standard global locations. This might be worth mentioning too.